internal/provider_test/integration_test.go

```diff
@@ -990,7 +997,7 @@ func TestImmutableAttributeChange(t *testing.T) {
 	// Change the DBA password again, this time with
 	// NUODB_CP_ALLOW_DESTRUCTIVE_REPLACE=true
 	t.Run("applyDbaPasswordChangeWithReplace", func(t *testing.T) {
-		os.Setenv(framework.ALLOW_DESTRUCTIVE_REPLACE_VAR, "true")
+		t.Setenv(framework.ALLOW_DESTRUCTIVE_REPLACE_VAR, "true")
 		defer os.Unsetenv(framework.ALLOW_DESTRUCTIVE_REPLACE_VAR)
```

Comment:
You can drop the `defer` on line 1001, right? It seems like `t.Setenv()` does the right thing.

Reply:
Also, should `t.Setenv()` be used on the negative checks that reset `NUODB_CP_URL_BASE`? I think this is what you were commenting on in that PR, but I didn't pick up on it.